src/shadowbox/server/server_config.ts

```diff
@@ -31,6 +31,10 @@ export interface ServerConfigJson {
   portForNewAccessKeys?: number;
   // Which staged rollouts we should force enabled or disabled.
   rollouts?: RolloutConfigJson[];
+  // The hostname for the proxy server.
+  hostname?: string;
+  // The port for the management API.
+  apiPort?: number;
```

Comment:
We need to think more about the end-to-end experience of changing the apiPort. Changing this field will not work as intended on DigitalOcean. And for manual servers it will break existing managers.

Let's revert the apiPort config for now.

Reply:
Done.